manual/de/ch_Tools_Assistants.xml

```diff
@@ -5115,13 +5117,13 @@
       <glossentry>
         <glossterm>Preis</glossterm>
         <glossdef>
-          <para>Kommt zwischen einer Nicht-Währung und einer Währung zur Anwendung.
+          <para>In einem Kauf oder Verkauf realisierter Kurs.
```

Comment:
Note: In der Umgangssprache und einigen &app;-Dialogen wird der Begriff weiter gefaßt, also auch dort verwendet, wo es sich strenggenommen um Kurse handelt.

Siehe Diskussion unten.